tests/services/inventory/update_cai_dumps.py

```diff
@@ -256,6 +256,11 @@ def forwardingrule(item):
     return _create_compute_asset(item, 'compute.googleapis.com/ForwardingRule')
 
 
+def globalforwardingrule(item):
```

Comment:
This file shouldn't need to be changed, this is needed when going from the mock API to the CAI dump, not the other way around.

Reply:
Gotcha. Reverted changes.